### PR TITLE
add release close to all request in sdc client

### PR DESCRIPTION
**Pull request by jesuspg**
@jframos @hmunfru

### CODE REVIEW

**Comment by flopezag** [comment]
LGTM

File: client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/impl/AbstractInstallableService.java [hunk 1]
```diff
@@ -56,9 +57,17 @@
...
-        Invocation.Builder builder = createWebResource(url, token, vdc);
-        builder = addCallback(builder, callback);
-        return builder.put(Entity.json(null), Task.class);
+        Response response = null;
+        try {
+            Invocation.Builder builder = createWebResource(url, token, vdc);
+            builder = addCallback(builder, callback);
+            response = builder.put(Entity.json(null));
+            return response.readEntity(Task.class);
+        } finally {
```
Comment (jframos) [inline client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/impl/AbstractInstallableService.java]: According that code, if all works right, the 'return' sentence in the 'try' block will give back the control to the caller function, and 'finally' block will not be executed, so 'response' will not be closed. Is that the expected behavior?

Comment (jesuspg) [inline client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/impl/AbstractInstallableService.java]: first of all, before return the control, finally block is executed. FYI: http://docs.oracle.com/javase/tutorial/essential/exceptions/finally.html

Comment (jframos) [inline client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/impl/AbstractInstallableService.java]: :scream: Very interesting! :dancer:

**Comment by jframos** [comment]
LGTM

**Pull request merged**